ts/src/pro/gemini.ts

```diff
@@ -500,12 +500,17 @@ export default class gemini extends geminiRest {
         currentBidAsk['timestamp'] = timestamp;
         currentBidAsk['datetime'] = this.iso8601 (timestamp);
         currentBidAsk['info'] = rawBidAskChanges;
+        const bidsAsksDict = {};
+        bidsAsksDict[symbol] = currentBidAsk;
         this.bidsasks[symbol] = currentBidAsk;
-        client.resolve (currentBidAsk, messageHash);
+        client.resolve (bidsAsksDict, messageHash);
```

Comment:
it was incorrectly returning just ticker object, instead of `{ BTC/USDT: tickerObj}`